ProcessLib/RichardsMechanics/CreateRichardsMechanicsProcess.cpp

```diff
@@ -134,14 +134,22 @@ std::unique_ptr<Process> createRichardsMechanicsProcess(
     auto media_map =
         MaterialPropertyLib::createMaterialSpatialDistributionMap(media, mesh);
 
-    std::array const requiredGasProperties = {MaterialPropertyLib::viscosity};
+    std::array const requiredMediumProperties = {
```

Comment:
```suggestion
    std::array const required_medium_properties = {
```
Same, as you have fixed above....